grc/core/generator/top_block.py

```diff
@@ -196,6 +196,14 @@ def _get_block_sort_text(block):
         ]
 
         blocks = expr_utils.sort_objects(blocks, operator.attrgetter('name'), _get_block_sort_text)
+
+        # Ordering blocks : blocks with GUI Hint must be processed first to avoid PyQT5 superposing blocks
+        for block in blocks:
+            hint = block.params.get('gui_hint')
+            block.without_gui_hint = (hint == None) or (not hint.get_value())
+
+        blocks = sorted(blocks, key=lambda block: block.without_gui_hint)
```

Comment:
This code is a little cumbersome, actually. May I suggest to something like this instead:
```python
def without_gui_hint(block):
    hint = block.params.get('gui_hint')
    return hint is None or not hint.get_value()

blocks.sort(key=without_gui_hint)
``` 
Now, if only we could use the walrus operator, then I'd be a one-liner =)